test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/SingleDisposalsTriageControllerSpec.scala

```diff
@@ -2746,10 +3195,115 @@ class SingleDisposalsTriageControllerSpec
         }
       }
     }
+  }
+
+  def displayIndividualCapacitorTriagePageBehaviorIncompleteJourney(performAction: () => Future[Result])(
+    requiredPreviousAnswers: IncompleteSingleDisposalTriageAnswers,
+    answersWithCurrentAnswer: IncompleteSingleDisposalTriageAnswers,
+    description: Option[String] = None
+  )(
+    pageTitleKey: String,
+    checkContent: Document => Unit,
+    checkPrepopulatedContent: Document => Unit
+  ): Unit = {
+    val scenarioDescription = description.map(_ + " and when ").getOrElse("")
+    s"display the page when ${scenarioDescription}no option has been selected before for capacitor" in {
+      List(
+        sessionDataWithStartingNewDraftReturn(requiredPreviousAnswers)._1,
+        sessionDataWithFillingOutReturn(requiredPreviousAnswers)._1
+      ).foreach { currentSession =>
+        withClue(s"For currentSession $currentSession: ") {
+          inSequence {
+            mockAuthWithNoRetrievals()
+            mockGetSession(currentSession)
+          }
+
+          checkPageIsDisplayed(
+            performAction(),
+            messageFromMessageKey(pageTitleKey),
+            checkContent
+          )
+        }
+      }
+    }
+
+    s"display the page when ${scenarioDescription}an option has been selected before for capacitor" in {
+      List(
+        sessionDataWithStartingNewDraftReturn(answersWithCurrentAnswer)._1,
+        sessionDataWithFillingOutReturn(answersWithCurrentAnswer)._1
+      ).foreach { currentSession =>
+        withClue(s"For currentSession $currentSession: ") {
+          inSequence {
+            mockAuthWithNoRetrievals()
+            mockGetSession(currentSession)
+          }
+
+          checkPageIsDisplayed(
+            performAction(),
+            messageFromMessageKey(pageTitleKey), { document =>
+              checkContent(document)
+              checkPrepopulatedContent(document)
+            }
+          )
+        }
+      }
+    }
+  }
+
+  def displayIndividualPersonalRepTriagePageBehaviorIncompleteJourney(performAction: () => Future[Result])(
```

Comment:
this function looks exactly the same as `displayIndividualCapacitorTriagePageBehaviorIncompleteJourney` just above - it makes sense to have a single method with a more generic name or if you wanted a method for personal reps and another for capacitors, it only makes sense to have separate ones if they actually do something meaningfully different.

Reply:
I tried to use same but getting duplicate test error. Will try to use same method